plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/MavenInvoker.java

```diff
@@ -223,6 +224,8 @@ private void handleInvocationResult(Plugin plugin, InvocationResult result) {
             if (result.getExecutionException() != null) {
                 LOG.error(plugin.getMarker(), "Execution exception occurred: ", result.getExecutionException());
                 plugin.addError(result.getExecutionException());
+            } else {
+                plugin.addError(new MavenInvocationException("Build failed with code: " + result.getExitCode()));
```

Comment:
In case the maven command finish but return a non-zero status code in case of build failure. Was ignored